unit-tests/py/rspy/unittest.py

```diff
@@ -336,6 +336,8 @@ def __init__( self, testname, exe ):
         :param exe: full path to executable
         """
         global unit_tests_dir
+        if not os.path.isfile( exe ):
+            log.f( "Tried to create exe test with invalid exe file:", exe )
```

Comment:
I think raising a RuntimeError would work better, no?

Reply:
That will cause the whole call stack to be printed. IMO that's not necessary but if you prefer I'll change it

Reply:
Programming runtime errors should show the call stack, yes: this is not an expected error, and we want as much info printed to help debug